cmd/api/src/api/tools/pg_test.go

```diff
@@ -0,0 +1,204 @@
+package tools_test
```

Comment:
Since this is an integration test please prefix it with:

```
//go:build integration
// +build integration

package tools_test
```

This will ensure that unit test runs do not invoke the test needlessly when devs are performing fast-pass validation of other testing work.